oojs.md

```diff
@@ -7,15 +7,30 @@ Define a `Playlist` class. It should have...
 - An `addSong` (Function) method that adds a song to the `songs` property
 
 ```js
-Your answer goes here
+var titleText = document.getElementById('title').value
+var songsText = document.getElementById('songs').value
+var songsArray = []
+var favorites = 0;
+
+class Playlist {
+  constructor(title, songs) {
+    this.title  = titleText;
+    while (songsText != ""){
+      songsArray.push(songsText);
+    }
```

Comment:
this while loop would run indefinitely in most cases, unless you set `songsText` to `""` after `.push`ing.

`this.songs = songs` would be enough, and you would 'assume' that `songs` would be an array